### PR TITLE
Nil Frame error patch

### PR DESCRIPTION
**Pull request by araddon**

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -396,6 +403,9 @@ func (c *Conn) recv() error {
 	}
 
 	call := &c.calls[head.stream]
+	if call == nil {
```
Comment (kyledj) [inline conn.go]: `call.framer` was the `nil` portion, right? `call` itself was not `nil`

Comment (schmichael) [inline conn.go]: I don't think this was the problem.

File: conn.go [hunk 2]
```diff
@@ -396,6 +403,9 @@ func (c *Conn) recv() error {
...
+		return fmt.Errorf("gocql: error on disappearing stream %d: %v", head.stream, frame)
+	}
 	err = call.framer.readFrame(&head)
```
Comment (mdmarek) [inline conn.go:409]: Would it be worthwhile to also `nil` check `call.framer` ?

**Comment by schmichael** [comment]
This was the nil: https://github.com/gocql/gocql/blob/master/frame.go#L369

**Comment by schmichael** [comment]
:+1:

**Comment by epsniff** [comment]
:+1:

**Pull request merged**